src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -85,12 +92,16 @@ const defaultSearchParams: PeopleSearchQuery = {
   major: '',
   minor: '',
   residence_hall: '',
-  class_year: '',
+  class_standing: '',
+  graduation_year: '',
   home_town: '',
   state: '',
   country: '',
   department: '',
   building: '',
+  //graduation_year_range: '',
```

Comment:
```suggestion
  //graduation_year_range: '',
```
Is this meant to be commented out?

Reply:
I forgot to remove this comment. Thanks for pointing it out!